source/support.html.erb

```diff
@@ -136,6 +136,15 @@ title: Support
         Outside of working hours, please use the escalation contact
         details you have been sent.
       </p>
-    </div>
+      <h2 class="heading-large">Keep up to date</h2>
+
+      <p>
+        Current GOV.UK PaaS <a href="https://status.cloud.service.gov.uk">platform status</a>. Subscribe here for incident alerts.
```

Comment:
I don't think we should tell tenants about the status page until the team know how and when to update it.